src/app/ConcreteAttributePath.h

```diff
@@ -37,9 +38,9 @@ struct ConcreteAttributePath
         return mEndpointId == other.mEndpointId && mClusterId == other.mClusterId && mAttributeId == other.mAttributeId;
     }
 
-    EndpointId mEndpointId   = 0;
-    ClusterId mClusterId     = 0;
-    AttributeId mAttributeId = 0;
+    EndpointId mEndpointId   = kInvalidEndpointId;
+    ClusterId mClusterId     = ClusterInfo::kInvalidClusterId;
+    AttributeId mAttributeId = ClusterInfo::kInvalidAttributeId;
```

Comment:
Again, I don't think we need this, if we change the `Valid()` bits above as I suggest.